.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: python
+dist: trusty
+python:
+  - "2.7"
+services:
+  - postrgresql
+before_install:
+  - sudo ./apt/production.sh
+  - sudo ./apt/testing.sh
+install:
+  - "pip install setuptools==32"
+  - "pip install -r requirements.txt"
+  - "pip install -r requirements/testing.txt"
+before_script:
+  - "psql -c 'CREATE DATABASE manchester_traffic_offences;' -U postgres"
```

Comment:
Is it worth enforcing the encoding at this point? It's probably utf-8 by default everywhere by now.